.github/workflows/setup.sh

```diff
@@ -4,7 +4,7 @@ set -ex
 
 cd src
 
-npm i --silent -g pnpm@5.1.7 esbuild@0.7.9 --unsafe-perm
-pnpm i --no-prefer-frozen-lockfile --reporter=silent --ignore-scripts
+npm i --silent -g pnpm@5.1.7 --unsafe-perm
+pnpm i --no-prefer-frozen-lockfile
 
```

Comment:
I guess that means we can now update the CONTIBUTING.md file and remove the install esbuild step 🎊